HW_001/task_004.py

```diff
@@ -0,0 +1,15 @@
+#Задача 4: 
+#Петя, Катя и Сережа делают из бумаги журавликов.
+# Вместе они сделали S журавликов. 
+#Сколько журавликов сделал каждый ребенок, если известно, 
+#что Петя и Сережа сделали одинаковое количество журавликов,
+#а Катя сделала в два раза больше журавликов,
+#чем Петя и Сережа вместе?
+#Пример
+#6 -> 1  4  1
+#24 -> 4  16  4
+#60 -> 10  40  10
+
+S = int(input('Введите сколько всего ребята сделали журавликов? : '))
+ps = int(str(S)) // 6
```

Comment:
Немного не понял. Переменная  S уже была в целочисленном типе данных. Зачем вы её переводите в строковый тип, чтобы потом опять перевести в целочисленный?

Reply:
Чтоб данные были в одной строке, экономия памяти.

Reply:
Так вы наоборот больше памяти потратили. S уже была в целом типе. Нужно было просто 
```suggestion
ps = S // 6
```